### PR TITLE
Image Offsets Swap

### PR DESCRIPTION
**Pull request by j-c-c**
This PR fixes our offsets, which were xy-swapped from that of Relion's.

See #1146 for more details.

### CODE REVIEW

File: src/aspire/image/image.py [hunk 1]
```diff
@@ -465,7 +465,7 @@ def _im_translate(self, shifts):
             xp.asarray(np.ceil(np.arange(-L / 2, L / 2, dtype=self.dtype)))
         )
         grid_1d = xp.asnumpy(grid_shifted) * 2 * np.pi / L
-        om_x, om_y = np.meshgrid(grid_1d, grid_1d, indexing="ij")
+        om_x, om_y = np.meshgrid(grid_1d, grid_1d, indexing="xy")
```
Comment (garrettwrong) [inline src/aspire/image/image.py]: Lets add a comment for the next developer. Thanks

**Review by garrettwrong — DISMISSED** [review]
Looks good, just the one comment request and should be ready to move on.  Thanks!

File: src/aspire/image/image.py (hunk 1 above)
Comment (j-c-c) [reply src/aspire/image/image.py]: Sure

**Review by garrettwrong — DISMISSED** [review]

File: tests/test_image.py [hunk 2]
```diff
@@ -112,14 +113,14 @@ def testImShiftStack(parity, dtype):
     im2 = ims.shift(shifts.astype(dtype))
     # ground truth numpy roll
     im3 = np.array(
-        [np.roll(ims_np[i, :, :], -shifts[i], axis=(0, 1)) for i in range(n)]
```
Comment (janden) [inline tests/test_image.py:115]: Can you add a comment here explaining the axis swap?

**Review by janden — CHANGES_REQUESTED** [review]
Looks good. Just one thing.

File: tests/test_image.py (hunk 2 above)
Comment (j-c-c) [reply tests/test_image.py:115]: Sure! Done.

**Review by garrettwrong — APPROVED** [review]

**Comment by garrettwrong** [comment]
I think you are probably good to merge this. Thanks

**Pull request merged**